aria-practices.html

```diff
@@ -6279,6 +6279,39 @@ <h3>Indicating sort order with <code>aria-sort</code></h3>
 
   </section>
 
+  <section id="accessibility_tree">
+    <h2>Accessibility tree</h2>
+    <p>Browsers construct an accessibility tree to communicate the representation of web content with assistive technologies. See the <a href="https://w3c.github.io/core-aam/#dfn-accessibility-tree">Core Accessibility API Mappings</a> specification for further information about accessibility trees.</p>
+    <p>The accessibility tree is based on information in the DOM tree (including ARIA) and information in CSS. For example, text in the DOM tree can also be in the accessibility tree. CSS generated content is not part of the DOM tree, but can be part of the accessibility tree.</p>
+    <p>For example, given the HTML document</p>
```

Comment:
```suggestion
    <p>For example, given the HTML document:</p>
```

colon after document, to introduce example code